assets/js/frontend/checkout.js

```diff
@@ -514,7 +514,7 @@ jQuery( function( $ ) {
 						wc_checkout_form.detachUnloadEventsOnSubmit();
 
 						try {
-							if ( 'success' === result.result && $form.triggerHandler( 'checkout_place_order_success' ) !== false ) {
+							if ( 'success' === result.result && $form.triggerHandler( 'checkout_place_order_success', [result, wc_checkout_form] ) !== false ) {
```

Comment:
We know the operation was successful here; I was interested to learn why we also pass `result` to any registered handlers?

Reply:
@barryhughes I know this PR is quite old now but a couple of questions:

1. Is it still a relevant change, that we could rebase and get merged?
2. If not can it be closed?

Thanks 🙇🏻

Reply:
Yes, Still we need this. 
If `wc_checkout_form` param available beside `result` param on `checkout_place_order` event, 
we can reuse `wc_checkout_form` methods like: `wc_checkout_form.blockOnSubmit` or `wc_checkout_form.attachUnloadEventsOnSubmit()` functions.